glean-core/uniffi/src/glean.udl

```diff
@@ -5,6 +5,17 @@ namespace glean {
     void enable_logging();
 
     void set_upload_enabled(boolean enabled);
+
+    // Experiment reporting API
+    void set_experiment_active(string experiment_id, string branch, record<DOMString, string> extra);
+    void set_experiment_inactive(string experiment_id);
+    boolean test_is_experiment_active(string experiment_id);
+    RecordedExperiment test_get_experiment_data(string experiment_id);
```

Comment:
I actually realised that we can now merge these together into this:

```suggestion
    RecordedExperiment? test_get_experiment_data(string experiment_id);
```

Then the SDK wrapper can just call that single function in both `testIs` and `testGet` and do the right thing depending on whether it got null or not.

Reply:
Nice! I don't think anyone is using the experiment test api aside from our internal tests right now, so I'd be okay with making this change even without the SDK wrapper, maybe with a comment that explains `null` = no experiment with that id.